### ISSUE TITLE
Split view - UI polish

### ISSUE THREAD

**Issue by aguscruiz** [brave/brave-browser#39027]
### Description

Latest version of Figma

https://www.figma.com/design/Dt0iJdVaaAj7DikWLO1P7I/Split-view-%2F-Sidebar-browsing?node-id=348-28191&t=0r7ExpW2INc0KU4d-1

### On hold
- [x] Update all colors to material color tokens

### Tab bar
- [x] #39335

<img width="676" alt="image" src="https://github.com/brave/brave-browser/assets/15734630/c8b88e51-6a7d-4abd-8b7f-b8aaf744f61b">

- [x] Check correct colors for the split view tabs container background in dark mode: (https://github.com/brave/brave-core/pull/27008)
https://www.figma.com/design/Dt0iJdVaaAj7DikWLO1P7I/Split-view-%2F-Sidebar-browsing?node-id=2492-48088&t=OXIs7NQGq8BpdKxU-1 

- [x] Separator between tabs in split view tab (https://github.com/brave/brave-core/pull/27008) 
- [x] Horizontal tabs: should be primitive/neutral/10 (https://github.com/brave/brave-core/pull/27008) 
- [x] Vertical tabs: should be primitive/neutral/10 (https://github.com/brave/brave-core/pull/27008)

![Image](https://github.com/user-attachments/assets/59e7f9e0-3d5e-4531-a28b-853a4f9365e4)

- [x] Update tab UI https://github.com/brave/brave-browser/issues/42913
<img width="400" alt="image" src="https://github.com/user-attachments/assets/6006d739-96b3-43d5-91f4-45cc1ee1de48">

### Page area
- [x] Border color around the active tab has been updated
- [x] Border color of inactive tab has also been updated
- [x] Border of active tabs is 2px
- [x] Inactive tabs don't have a shadow
- [x] #39152
      https://github.com/brave/reviews/issues/1646 Figma link to spec: 
      https://www.figma.com/design/Dt0iJdVaaAj7DikWLO1P7I/Split-view-%2F-Sidebar-browsing?node-id=348-28191&t=0r7ExpW2INc0KU4d-1

### Contextual menu on gutter
- [x] When hovering, don't change the background color.
- [x] When active, change icon color to semantic/icon/interactive
- [x] The contextual menu itself needs a 4px padding on top-bottom to not make it look so cramped

**Comment by sangwoo108** [comment]
> Border of active tabs is 1.5px

@aguscruiz real number may not work on Windows. Can we have integer for the thickness?

**Comment by aguscruiz** [comment]
> @aguscruiz real number may not work on Windows. Can we have integer for the thickness?

Yeah we can use 2px instead. We can use that on macOS/Linux aswell, so we keep the same in all. Btw the color token I used is for border/icon/interactive. That should solve the issue with the color not having contrast in dark mode

> And curious if Browser/ToolBar/Button/Outline is valid token. I couldn't find the color.

This one is part of the new color tokens for the material design work. It's built on top of the new primitives, so it's not accessible without those. 

You need it for the url indicator?
<img width="484" alt="image" src="https://github.com/brave/brave-browser/assets/15734630/0cdaacb6-6584-470f-b2a4-47e1d7f4727b">

If that's the use case, you can use the same color we give the top tab bar background. It's the same value.
<img width="1473" alt="image" src="https://github.com/brave/brave-browser/assets/15734630/2aaf7b13-5670-408b-92d0-c5714c22e213">


I left a clean design without images here so you can see it better
https://www.figma.com/design/Dt0iJdVaaAj7DikWLO1P7I/Split-view-%2F-Sidebar-browsing?node-id=2318-16574&t=6FHr3SRo0nZmN0Fo-1

**Comment by sangwoo108** [comment]
> If that's the use case, you can use the same color we give the top tab bar background. It's the same value.

Yeah, I needed it for inactive web view's border. I guess that's the same value with the mini url bar's border.

**Comment by sangwoo108** [comment]
Browser's current frame color looks lighter than the design guide, so vaguely notice the border 🤔

**Comment by aguscruiz** [comment]
Yeah, another update from the material design Token update that's pending.

**Comment by sangwoo108** [comment]
Okay then let me push as is 👍

**Comment by TEMP-ad** [comment]
I recorded this yesterday but still applies to 1.69.66 Chromium: 127.0.6533.26 (Official Build) nightly (64-bit) 

Seems like this behavior appeared after `Polish Split view UI brave/brave-core#24385` PR, and it is basically, that when you click on the tabs of the split view, the active tab will have 2px less, than when it is not the active one.

You can see it in devtools, how the size changes.
Plus seems like devtools now disappears for a while when switching from non-devtools split view tab to the devtools one

And as can be seen, it happens in both VT and HT.

https://github.com/brave/brave-browser/assets/156715894/14e0ea9b-af65-4199-9a5a-90768aa2f7c1

https://github.com/brave/brave-browser/assets/156715894/3eb65f02-bdf1-4c46-ad3d-fa55b369bfdf

**Comment by rebron** [comment]
Marking close/completed. META issue for Split View - UI polish.
